hub/compute/tests/test_ray.py

```diff
@@ -18,6 +19,79 @@
 }
 
```

Comment:
We need to add tests for composite transforms here as well.

Reply:
agree, but it looks to mee it is gonna treat all transform functions as ray if .store is called on ray one.